dddpy/controllers/todo/todo_controller.py

```
@@ -0,0 +1,221 @@
+                                               get_find_todos_usecase,
+                                               get_start_todo_usecase,
+                                               get_update_todo_usecase)
+from dddpy.usecase.todo import (CompleteTodoUseCase, CreateTodoUseCase,
+                                FindTodoByIdUseCase, FindTodosUseCase,
+                                StartTodoUseCase, UpdateTodoUseCase)
+
+
+class TodoController:
+    """Controller class for handling Todo-related HTTP endpoints."""
+
+    def register_routes(self, app: FastAPI):
```

Ruff (0.8.2)

`register_routes` is too complex (22 > 10)

(C901)